driver/src/main/java/com/code_intelligence/jazzer/junit/JazzerFuzzTestExecutor.java

```diff
@@ -78,47 +78,37 @@ public TestExecutionResult execute() throws IOException, URISyntaxException {
     // libFuzzer and also emit findings into it so that the regression test can be used to debug
     // them.
     FuzzTest fuzzTest = AnnotationSupport.findAnnotation(fuzzTestMethod, FuzzTest.class).get();
-    String seedCorpusResourcePath = fuzzTest.seedCorpus().isEmpty()
-        ? Utils.defaultSeedCorpusPath(fuzzTestClass)
-        : fuzzTest.seedCorpus();
+    String seedCorpusResourcePath = Utils.inputsDirectoryResourcePath(fuzzTestClass);
```

Comment:
Rename variable?

Reply:
Done, also updated the comments and other variables.